requirements.txt

```diff
@@ -1,4 +1,3 @@
 beautifulsoup4==4.8.1
 esprima==4.0.1
-soupsieve==1.9.5
-click==7.0
\ No newline at end of file
+soupsieve==1.9.5
```

Comment:
Not really all too familiar with this stuff but it looks like we are still importing click so curious as to why it is no longer a requirement? Do we know it's already there because it's a requirement in cci?